test/unit/app/messaging/email/test-helpers/submission-agent.json

```diff
@@ -54,20 +54,27 @@
     "potentialFunding": "40000.00",
     "remainingCost": "60000.00",
     "consentOptional": true,
-    "storage": "boop",
-    "solarPVCost": null,
+    "solarPVCost": "",
     "smallerAbattoir": "No",
-    "IsSmallerAbattoir": false,
+    "fruitStorage": "Yes",
+    "IsSmallerAbattoirYes": false,
     "isMechanisationYes": true,
     "isFruitStorageTrue": true,
-    "fruitStorage": "Yes",
-    "otherFarmers": null,
+    "otherFarmers": "",
     "solarPVSystem": "No",
     "isSolarPVSystemYes": false,
-    "solarGrantRate": null,
+    "solarGrantRate": "",
     "grantRate": "up to 40%",
     "applicantBusiness": "why",
     "mechanisation": "Yes",
-    "manualLabour": "Up to 5% of workforce"
+    "manualLabour": "Up to 5% of workforce",
+    "howAddingValue" : "",
+    "productsProcessed" : "",
```

Comment:
Things like this should be deleted from here or set to null, to show yar key does not exist